src/components/DashboardBody.js

```diff
@@ -60,13 +16,13 @@ const BuildWidgets = widgetArray =>
 
 class DashboardBody extends Component {
   render() {
-    const renderWidgets = BuildWidgets(dashboardBodyProps.widgets)
+    const renderWidgets = buildWidgets( this.props.widgets )
```

Comment:
I would prefer to see this as a method in this Component, but since we'll be removing later, its ok to leave for now.